hil-test/tests/embassy_interrupt_spi_dma.rs

```diff
@@ -137,7 +137,7 @@ mod test {
         let other_peripheral = Spi::new_with_config(
             peripherals.SPI3,
             Config {
-                frequency: 100.kHz(),
+                frequency: 10000.kHz(),
```

Comment:
Had to pull this up: https://github.com/esp-rs/esp-hal/pull/2543#discussion_r1843613723